### PR TITLE
Update css_quick_guide_mode.md

### PR DESCRIPTION
**Pull request by hitesh-wagento**
## This PR is a: Fix typos
- [ ] New topic
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
Fix typos

When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_mode.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The following is an illustration of how the process of making simple changes loo
 
     ![Less code redefining the color of the primary buttons]({{ site.baseurl }}/common/images/extend_less_screenshot121.png)
 
-1. Change the button font by adding the following code in the `_extend.less` file:
+1. Change the button font size by adding the following code in the `_extend.less` file:
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_mode.md]: Please add these changes to v2.2 as well, as v2.3 is symlinked. This way, all versions will get the change.

**Comment by shrielenee** [comment]
Also, please ensure you fill out our PR form thoroughly, including check boxes in the first section.

File: guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_mode.md (hunk 1 above)
Comment (shrielenee) [reply guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_mode.md]: Your PR, #2179, should be merged with this one.

**Comment by hitesh-wagento** [comment]
Hi @shrielenee I have merged #2179 to #2178

**Pull request merged**